src/backend/distributed/executor/multi_utility.c

```diff
@@ -198,6 +198,26 @@ multi_ProcessUtility(Node *parsetree,
 				ErrorIfDistributedRenameStmt(renameStmt);
 			}
 		}
+
+		/*
+		 * ALTER ... SET SCHEMA statements have their node type as AlterObjectSchemaStmt
+		 * So, we intercept AlterObjectSchemaStmt to tackle these commands.
+		 */
+		if (IsA(parsetree, AlterObjectSchemaStmt))
+		{
+			ereport(NOTICE, (errmsg("Citus does not support ALTER ... SET SCHEMA")));
```

Comment:
We can use a WARNING instead of NOTICE. And may we can use a similar message to this one;
```
		ereport(NOTICE, (errmsg("not propagating CREATE ROLE/USER commands to worker"
								" nodes"),
						 errhint("Connect to worker nodes directly to manually create all"
								 " necessary users and roles.")));
```